package.json

```diff
@@ -169,10 +168,9 @@
     "eslint-config-stylelint": "^13.1.1",
     "got": "^11.8.2",
     "husky": "^6.0.0",
-    "jest": "^26.6.3",
-    "jest-circus": "^26.6.3",
-    "jest-preset-stylelint": "^4.1.0",
-    "jest-watch-typeahead": "^0.6.3",
+    "jest": "^27.0.1",
+    "jest-preset-stylelint": "github:stylelint/jest-preset-stylelint#jest-27",
+    "jest-watch-typeahead": "github:jest-community/jest-watch-typeahead",
```

Comment:
[note] Wait for a new version of `jest-watch-typeahead`:
https://github.com/jest-community/jest-watch-typeahead/issues/61

Reply:
Done. 369f4d83

https://github.com/jest-community/jest-watch-typeahead/releases/tag/v0.6.4